docs/welcome/getting-started.md

```diff
@@ -1,21 +1,84 @@
-# Getting started
+# Getting started with Asana
+
+In order to ensure we deliver your course, project, or practice content as successfully and effectively as possible, we use Asana, a project management tool, to track all the work involved. This way, all work is organized, tracked, and transparent to all.
+
+### So, how do I get started?
+
+1. Check your email inbox. 
+
+You should receive an invitation to join Asana from DataCamp. Follow the instructions to join your particular course/project/practice's project. Click on 'View Project'.
+
+![screenshot1](https://github.com/datacamp/authoring/blob/Asana-instructions/Welcome%20Email.png)
```

Comment:
The live version of the site doesn't have access to GitHub, AFAIK. You need to put the PNG files in the `images` directory, then add a link like `/images/your-image-dir/your-image.png`. See https://raw.githubusercontent.com/datacamp/authoring/master/docs/welcome/getting-started.md for an example.

Reply:
Where's the images directory? @richierocks